### PR TITLE
Principles of Management Recipe

### PR DESCRIPTION
**Pull request by TxsAdamWest**
🛠 WIP 🛠 

~~Currently working on creating style guide and creating snippets for Principles of Management since the raw file isn't available yet until new development server is created for TBD's. Once dev server is created, will be able to fetch raw for Principles of Management and begin book recipe development.~~  

- devb server is now up!  

- [x] Add bookmap requirements to Testy McTestFace - CTE Assignable Webview Ruleset/Recipe bookmaps:  https://docs.google.com/spreadsheets/d/1QM28dLjuEH3xPtVty9l7Zfeqp4mg4fJtIAzv-oXH5Ik/edit#gid=1173156174

- [x] Compare to Business Series Testing & requirements: https://docs.google.com/spreadsheets/d/1CY56IPhwap2aR_t2VQYsorOMp_8AxLaUiGywtaPscNs/edit#gid=257491970

- [x] Create Style Guide
- [x] Obtain raw for Principles of Management
- [x] Create Recipe -> comparing to generated PDF & bug fixing

### CODE REVIEW

**Comment by TxsAdamWest** [comment]
[Finalize recipe for Principles of Management](https://trello.com/c/83VCbvB6/347-finalize-recipe-for-principles-of-management)

File: books.txt [hunk 1]
```diff
@@ -26,26 +27,27 @@ declare -x -r RECIPE_DESC=(
...
-  "TEAapphysics       TEAap-physics     df721c85-4c66-4b94-9197-a386df683af5    tea.cnx.org          mixins/,books/ap-physics/,books/physics/"
-  "TEAapphysics2      TEAap-physics     12af895f-1ec3-4253-83d1-05724ea13273    tea.cnx.org          mixins/,books/ap-physics/,books/physics/"
-  "TEAmacroeconap     TEAeconomics      b30d51fa-86b7-4850-ae82-09a4428ff004    tea.cnx.org          mixins/,books/economics/"
-  "TEAmicroeconap     TEAeconomics      206e821b-0ef4-48dc-974d-3bbb635df3a4    tea.cnx.org          mixins/,books/economics/"
-  "TEAhsphysics       TEAhs-physics     43e35fe7-5afb-496d-b1b0-2f4bd7aab1f2    tea.cnx.org          mixins/,books/hs-physics/"
-  "TEAhsstats         TEAstatistics     b1258494-63dd-4a72-b0f0-a09e659964b7    tea.cnx.org          mixins/,books/statistics/"
-  "katalyst-physics   u-physics         4f45f6e1-a937-4390-a57a-a775b4065106    content01.cnx.org    mixins"
+# "_example           _example          example-uuid                            tea.cnx.org                   mixins/"
+  "statistics         statistics        30189442-6998-4686-ac05-ed152b91b9de    legacy-devb.cnx.org           mixins/"
```
Comment (helenemccarron) [inline books.txt]: Can we make the changes to all the books in a separate PR containing only `books.txt`? Also, the data is pulled from `devb.cnx.org`, not `legacy-devb.cnx.org `

Comment (TxsAdamWest) [reply books.txt]: Thanks! I'll fix that.  Not sure how to revert changes to a single file from a commit though..  I'll create another PR for the books.txt change.

File: books.txt [hunk 2]
```diff
@@ -27,28 +27,28 @@ declare -x -r RECIPE_DESC=(
 # - additionalStyleguideDirs : These specify which additional directories should be included when generating the styleguide
 
 BOOK_CONFIGS=(
-# "_example                   _example                   example-uuid                              tea.cnx.org            mixins/"
+# "_example           _example          example-uuid                            tea.cnx.org          mixins/"
   "principles-management      principles-management      62710222-02f1-4972-ab05-4060032e71f4      devb.cnx.org           mixins/"
-  "intro-business             intro-business             326ab45b-ef3c-4fdb-a4d4-a30c61b0b18b      devb.cnx.org           mixins/"
```
Comment (helenemccarron) [inline books.txt]: the server needs to be devb, that's where the collections are. Dev doesn't have the proper data

File: books.txt [hunk 3]
```diff
@@ -27,27 +27,28 @@ declare -x -r RECIPE_DESC=(
...
+  "statistics         statistics        30189442-6998-4686-ac05-ed152b91b9de    devb.cnx.org         mixins/"
+  "econ               economics         69619d2b-68f0-44b0-b074-a9b2bf90b2c6    dev.cnx.org          mixins/"
+  "macroecon          economics         4061c832-098e-4b3c-a1d9-7eb593a2cb31    dev.cnx.org          mixins/"
+  "macroeconap        economics         33076054-ec1d-4417-8824-ce354efe42d0    dev.cnx.org          mixins/"
+  "microecon          economics         ea2f225e-6063-41ca-bcd8-36482e15ef65    dev.cnx.org          mixins/"
+  "microeconap        economics         ca344e2d-6731-43cd-b851-a7b3aa0b37aa    dev.cnx.org          mixins/"
+  "biology            biology           185cbf87-c72e-48f5-b51e-f14f21b5eabd    dev.cnx.org          mixins/"
+  "apbiology          ap-biology        27cf60e4-bd82-418a-a7bb-a939c40f8f50    tea.cnx.org          mixins/,books/biology"
+  "physics            physics           031da8d3-b525-429c-80cf-6c8ed997733a    dev.cnx.org          mixins/"
+  "apphysics          ap-physics        8d04a686-d5e8-4798-a27d-c608e4d0e187    dev.cnx.org          mixins/,books/physics/"
```
Comment (helenemccarron) [inline books.txt]: all the older books should be pointing at devb, not dev

File: build/lib/cnxrecipes/__init__.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import json
```
Comment (helenemccarron) [inline build/lib/cnxrecipes/__init__.py]: please remove from this PR (not sure where this comes from)

File: cnx_recipes.egg-info/PKG-INFO [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
```
Comment (helenemccarron) [inline cnx_recipes.egg-info/PKG-INFO]: please remove from this PR

File: cnx_recipes.egg-info/SOURCES.txt [hunk 6]
```diff
@@ -0,0 +1,28 @@
+LICENSE
```
Comment (helenemccarron) [inline cnx_recipes.egg-info/SOURCES.txt]: please remove from this PR (not sure where this comes from)

File: recipes/output/manifest.json [hunk 7]
```diff
@@ -1,9 +1,4 @@
 [
```
Comment (helenemccarron) [inline recipes/output/manifest.json]: why is this removed? It is needed for intro-business (the merge conflict might solve this)

File: build/lib/cnxrecipes/recipes/manifest.json [hunk 8]
```diff
@@ -0,0 +1,66 @@
+[
```
Comment (helenemccarron) [inline build/lib/cnxrecipes/recipes/manifest.json]: please remove from this PR (not sure where this comes from)
